packages/cli/internal/pkg/cli/logs_workflow.go

```diff
@@ -193,6 +193,7 @@ If the --run flag is omitted then the latest workflow run is used.`,
 			}
 			return nil
 		}),
+		ValidArgsFunction: WorkflowAutoComplete,
```

Comment:
Does this also enable auto completion for the command?

Reply:
Yup, `ValidArgs` enables autocompletion if the completion is a noun, `RegisterFlagCompletionFunc` enables the completion if the completion is a flag